packages/cli/src/deprecation/deprecation.service.ts

```diff
@@ -39,25 +42,30 @@ export class DeprecationService {
 			envVar: 'N8N_SKIP_WEBHOOK_DEREGISTRATION_SHUTDOWN',
 			message: `n8n no longer deregisters webhooks at startup and shutdown. ${SAFE_TO_REMOVE}`,
 		},
+		{
+			envVar: 'N8N_RUNNERS_ENABLED',
+			message:
+				'Running n8n without task runners is deprecated. Please set `N8N_RUNNERS_ENABLED=true` to enable task runners. Learn more: https://docs.n8n.io/hosting/configuration/task-runners/',
+			checkValue: (value?: string) => value !== 'true',
```

Comment:
In config we also allow `1` and the value is not case sensitive. There's a bit of a concept mismatch here, as we are not deprecating an environment variable (what this service is for). We are deprecating a configuration (n8n w/o task runner)

Reply:
Since it's called `DeprecationService` I think it fits for _any_ deprecations, i.e. of env vars or of values of env vars. We're already doing this for `DB_TYPE`.